cmd/blockchaincmd/create.go

```diff
@@ -422,3 +436,43 @@ func checkInvalidSubnetNames(name string) error {
 	}
 	return nil
 }
+
+// TODO: add explain the difference for different validator management type
+func promptValidatorManagementType(
+	app *application.Avalanche,
+	sidecar *models.Sidecar,
+) error {
+	proofOfAuthorityOption := "Proof of Authority"
+	proofOfStakeOption := "Proof of Stake"
+	explainOption := "Explain the difference"
+	if createFlags.proofOfStake {
+		sidecar.ValidatorManagement = models.ValidatorManagementTypeFromString(proofOfStakeOption)
+		return nil
+	}
+	if createFlags.proofOfAuthority {
+		sidecar.ValidatorManagement = models.ValidatorManagementTypeFromString(proofOfAuthorityOption)
+		return nil
+	}
+	options := []string{proofOfAuthorityOption, proofOfStakeOption, explainOption}
```

Comment:
you need also to ask if the user wants to use or not SoV (that is the new ACP77 or the previous subnets).
or have SoV as a default but at least add a flag for it (most probably have a prompt?)

Reply:
this before asking for validator management protocol for ACP77